keyboards/tkw/grandiceps/config.h

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2020 Thys de Wet
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+#undef USE_I2C
+#undef SSD1306OLED
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x7812
+#define DEVICE_VER 0x0001
+#define MANUFACTURER TKW
+#define PRODUCT Grandiceps Split
+
+/* key matrix size */
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+
+
+
+#define MATRIX_COL_PINS { B0, A7, A3, A5, A4, A2 }
+#define MATRIX_ROW_PINS { B12, A6, B13, B9, B8 }
+#define MATRIX_COL_PINS_RIGHT { B0, A7, A3, A5, A4, A2 }
+#define MATRIX_ROW_PINS_RIGHT  { B12, A6, B13, B9, B8 }
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+#define MASTER_RIGHT
```

Comment:
I'm not a fan of having this defined by default.  No master set means it defaults to master left, and allows users to specify other options.